Add notes regarding optional dependencies (#2826)
pyvista/pyvista: doc/getting-started/installation.rst

* `NumPy <https://pypi.org/project/numpy/>`_ - NumPy arrays provide a core foundation for PyVista's data array access.
* `imageio <https://pypi.org/project/imageio/>`_ - This library is used for saving screenshots.
* `appdirs <https://pypi.org/project/appdirs/>`_ - Data management for our example datasets so users can download tutorials on the fly.
* `scooby <https://github.com/banesullivan/scooby>`_ - Reporting and debugging tools.

Optional Dependencies
~~~~~~~~~~~~~~~~~~~~~

> PyVista includes several optional dependencies for visualization and file IO, including:

Reviewer: ```suggestion
PyVista includes several optional dependencies for visualization and files IO, including:
```

Author: ```suggestion
PyVista includes several optional dependencies for visualization and reading a variety of additional file formats, including:
```

    pip install pyvista

> To install all the additional packages that extend pyvista, install using

Reviewer: ```suggestion
To install all the additional packages that extend PyVista, install using
```
